scst-store/amr/graphql-query.hbs.md

```diff
@@ -0,0 +1,181 @@
+see [Ingress support for Supply Chain Security Tools - Store](../ingress.hbs.md).
+
+### <a id='amr-graphql-access-token'></a> Retrieving the AMR GraphQL Access Token
+
+When you access the AMR GraphQL by using the GraphQL playground or cURL, you
+must retrieve the access token.
+
+To fetch the token, run:
+
+```console
+kubectl -n metadata-store get secret amr-graphql-view-token -o json | jq -r ".data.token" | base64 -d
+```
+
+### <a id='connect-graphql-playground'></a> Connecting to AMR GraphQL playground
+
+To connect to the AMR GraphQL playground when you enabled ingress, visit
+`https://amr-graphql.INGRESS-DOMAIN/play`.
+
+Where `INGRESS-DOMAIN` is the domain of the ingress you want to use.
+
+In the **Headers** tab at the bottom of the query window, add a JSON block containing the following authentication header:
+
+```json
+{
+  "Authorization": "Bearer ACCESS-TOKEN"
+}
+```
+
+Where `ACCESS-TOKEN` is the AMR GraphQL access token.
+
+You can use this to write and execute your own GraphQL queries to fetch data from the AMR.
+
+### <a id='connecting-to-graphql-curl'></a> Connecting to AMR GraphQL through cURL
+
+To connect to the AMR GraphQL using cURL when you enabled ingress, you first need the AMR GraphQL
+access token and its CA certificate.
+
+To fetch the AMR GraphQL CA certificate:
+
+```console
+kubectl get secret amr-app-tls-cert -n metadata-store -o json | jq -r '.data."ca.crt"' | base64 -d > /tmp/graphql-ca.crt
+```
+
+After the token and certificate are retrieved, you can use cURL to perform GraphQL queries by using the
+`https://amr-graphql.INGRESS-DOMAIN/query` endpoint.
+
+Where `INGRESS-DOMAIN` is the domain of the ingress you want to use.
+
+For example:
+
+```console
+curl "https://amr-graphql.<ingress-domain>/query" \
+  --cacert /tmp/graphql-ca.crt \
+  -H "Authorization: Bearer ACCESS-TOKEN" \
+  -H 'accept: application/json' \
+  -H 'content-type: application/json' \
+  --data-raw '{"query":"query getAppAcceleratorRuns { appAcceleratorRuns(first: 250){ nodes { guid name namespace timestamp } pageInfo{ endCursor hasNextPage } } }"}' | jq .
+```
+
+Where:
+
+- `ACCESS-TOKEN` is the AMR GraphQL access token
+- `/tmp/graphql-ca.crt` is the file location containing the AMR GraphQL CA certificate
+
+You can use this to write and execute your own GraphQL queries to fetch data from the AMR.
+
+This section uses cURL to query the AMR GraphQL endpoint, but you can use other similar tools to access the endpoint
+and provide them with the AMR GraphQL access token and CA certificate.
+
+## <a id='query-app-accelerator-runs'></a> Querying for AppAcceleratorRuns (alpha)
+
+This section tells you about GraphQL query arguments, and lists the fields available for `AppAcceleratorRuns` and `AppAcceleratorFragments`.
+
+### <a id='app-accelerator-runs-query-args'></a> AppAcceleratorRuns query arguments
+
+(Optional) You can specify the following supported arguments when querying for `AppAcceleratorRuns`.
```

Comment:
What are you indicating is Optional here?
Below it says  - You must specify at least one field.

Reply:
The arguments are optional - so a user can choose to not specify any arguments and the query will then return all AppAcceleratorRuns

The user must specify at least one field or the query will not know what data to return.